File: Tests/AINotebookCoreTests/AppLanguageTests.swift
import XCTest
@testable import AINotebookCore

final class AppLanguageTests: XCTestCase {
    func testAllCases() {
        XCTAssertEqual(AppLanguage.allCases, [.english, .czech])
    }

    func testRawValues() {
        XCTAssertEqual(AppLanguage.english.rawValue, "en")
        XCTAssertEqual(AppLanguage.czech.rawValue, "cs")
    }

    func testDisplayNames() {
        XCTAssertEqual(AppLanguage.english.displayName, "English")
        XCTAssertEqual(AppLanguage.czech.displayName, "Čeština")
    }
}

File: windows/tests/AINotebook.Core.Tests/AppLanguageTests.cs
using AINotebook.Core.Models;
using Xunit;

namespace AINotebook.Core.Tests;

public class AppLanguageTests
{
    [Fact] // testAllCases
    public void AllCases() =>
        Assert.Equal(new[] { AppLanguage.English, AppLanguage.Czech }, Enum.GetValues<AppLanguage>());

    [Fact] // testRawValues
    public void RawValues()
    {
        Assert.Equal("en", AppLanguageExtensions.RawValue(AppLanguage.English));
        Assert.Equal("cs", AppLanguageExtensions.RawValue(AppLanguage.Czech));
    }

    [Fact] // testDisplayNames
    public void DisplayNames()
    {
        Assert.Equal("English", AppLanguageExtensions.DisplayName(AppLanguage.English));
        Assert.Equal("Čeština", AppLanguageExtensions.DisplayName(AppLanguage.Czech));
    }
}
